articles/connections/social/oauth2.md

```diff
@@ -14,199 +14,11 @@ useCase:
     - customize-connections
     - add-idp
 ---
-# Connect Apps to Generic OAuth2 Authorization Servers
```

Comment:
This is for a custom connection and is for the extension so probably another one to omit